### ISSUE TITLE
Newsletter 26

### ISSUE THREAD

**Issue by ozkriff** [rust-gamedev/rust-gamedev.github.io#783]
This is a placeholder issue for September's Rust gamedev newsletter - it will be updated with the outline and contribution instructions closer to the end of the month. If you want to submit a section earlier than that, please feel free!

**Comment by yeahross0** [comment]
I have an update for Weegames.

**Comment by PsichiX** [comment]
taking `emergent`

**Comment by AngelOnFira** [comment]
I'll take Garden :rose:

**Comment by ozkriff** [comment]
Yeah, Garden is now known as Idu. I'll collapse the sections together.

I'm not 100% sure, but I vaguely recall that @logicsoup mentioned on DIscord that they wants to write it? It may be worth to clarify this.

**Comment by AngelOnFira** [comment]
Sounds good, I'll pull `Slides from GDC talk by Dan Olson` in the meantime.

**Comment by MoleTrooper** [comment]
I'll add a bit about Starframe again :wave:

**Comment by dabreegster** [comment]
I'm overloaded by project management; no A/B Street update, sorry.

**Comment by johanhelsing** [comment]
I'll write about Matchbox :)

**Comment by LechintanTudor** [comment]
I've written about Sparsey

**Comment by ManevilleF** [comment]
I can write about `bevy_verlet`

**Comment by mkiss-crey** [comment]
I'll write about Molecoole

**Comment by Hilku** [comment]
I'll write about Molecoole

**Comment by cwfitzgerald** [comment]
Taking `rend3` and the new `ktx2`. `imgui-wgpu` isn't needed, it's just a version bump update.

**Comment by Keavon** [comment]
No update for Graphite this month, @TrueDoctor and I have been fully consumed by school but that should hopefully change pretty soon as we get back to regularly scheduled development progress and updates. We'll also try and opt in for [Hacktoberfest](https://hacktoberfest.digitalocean.com/) if any folks want to contribute and get a free T-shirt. Maybe if someone would be willing to post a line at the bottom of the newsletter, in the Help Wanted section, that the Graphite project is looking for contributors to help reach the 0.1 Alpha release and we're participating as a Hacktoberfest project. I'd appreciate that! Thanks.

**Comment by mrDIMAS** [comment]
Taking rg3d, not sure if I can do it today though.

**Comment by AngelOnFira** [comment]
Taking Borderlands 3 Save Editor :smiley:

**Comment by olefasting** [comment]
Sorry for the late reply. I have been working on FishGame this month and been very busy with the new level editor. I'll have to wait until next month with a submission, which will probably be for the best, any ways, as I am hoping to release it to crates.io, around then (which means the API will be more or less determined)

**Comment by Ralith** [comment]
Took hecs.

**Comment by Dooskington** [comment]
I can throw together a section for Antorum Online today, thanks for the ping

**Comment by ozkriff** [comment]
Also taking "pixels 0.6" and "KAS GUI v0.10".

**Comment by ozkriff** [comment]
Final PR - https://github.com/rust-gamedev/rust-gamedev.github.io/pull/816 - everyone is invited to review. Merging in an hour if there're no objections.

**Comment by ozkriff** [comment]
And it's out! https://gamedev.rs/news/026 :tada: 

As always, huge thanks to all editors, contributors, and reviewers! :crab: :crab: :crab:
